Use "media" interface instead of "video" (deprecated in libquvi-0.2.15,
removed in 0.2.20). Fixes compatibility with libquvi-0.4.

See https://bugs.gentoo.org/show_bug.cgi?id=386651,
https://bugzilla.gnome.org/show_bug.cgi?id=661451

--- a/plparse/totem-pl-parser-videosite.c
+++ b/plparse/totem-pl-parser-videosite.c
@@ -75,7 +75,7 @@
 #ifdef HAVE_QUVI
 	QUVIcode rc;
 	quvi_t handle;
-	quvi_video_t v;
+	quvi_media_t v;
 	char *uri;
 	/* properties */
 	const char *video_uri;
@@ -104,8 +104,8 @@
 		return TOTEM_PL_PARSER_RESULT_ERROR;
 	}
 
-	getprop (QUVIPROP_VIDEOURL, video_uri);
-	if (quvi_getprop (v, QUVIPROP_VIDEOFILELENGTH, &length) == QUVI_OK)
+	getprop (QUVIPROP_MEDIAURL, video_uri);
+	if (quvi_getprop (v, QUVIPROP_MEDIACONTENTLENGTH, &length) == QUVI_OK)
 		length_str = g_strdup_printf ("%f", length);
 	else
 		length_str = NULL;

